### DISCUSSION TITLE
How to delete Stored Procedures?

### DISCUSSION THREAD

**Discussion by ru88s** [supabase/supabase#1338] [Questions] [answered]
![スクリーンショット 2021-05-01 15 27 51](https://user-images.githubusercontent.com/42270511/116773674-cc547900-aa91-11eb-849d-5747bb689c3b.png)

I want to delete the funciton.
What should I do?

**Comment by inian** [comment, marked as answer]
run this in the sql editor
```
DROP FUNCTION IF EXISTS incrementlike(int);
```
You might need to replace int above the proper parameter types.

**Reply by ru88s** [reply]
`DROP FUNCTION IF EXISTS incrementlike(x int, row_id int);`

I was able to delete!
Thank so much!

**Comment by hjaber** [comment]
I had a similar issue but had objects that depended on the function which required adding the word cascade:

`DROP FUNCTION IF EXISTS incrementlike CASCADE;`
